test/services/confirmation.statement.service.unit.ts

```diff
@@ -0,0 +1,33 @@
+jest.mock("private-api-sdk-node/dist/services/confirmation-statement");
+jest.mock("private-api-sdk-node/");
+
+import { getSessionRequest } from "../mocks/session.mock";
+import { createConfirmationStatement }
+  from "../../src/services/confirmation.statement.service";
+import { ConfirmationStatementService }
+  from "private-api-sdk-node/dist/services/confirmation-statement";
+import PrivateApiClient from "private-api-sdk-node/dist/client";
+import { createPrivateApiClient } from "private-api-sdk-node";
+
+const mockPostNewConfirmationStatement
+    = ConfirmationStatementService.prototype.postNewConfirmationStatement as jest.Mock;
+const mockCreatePrivateApiClient = createPrivateApiClient as jest.Mock;
+
+const transactionId = "12345";
+
+describe ("Confirmation statement api service unit tests", () => {
+
+  beforeEach (() => {
+    mockCreatePrivateApiClient.mockReturnValue({
+      confirmationStatementService: ConfirmationStatementService.prototype
+    } as PrivateApiClient
+    );
+  });
+
+  it ("should create confirmation statement is called in the private sdk", async () => {
```

Comment:
"should create confirmation statement is called in the private sdk" I think is meant to say "should call create confirmation statement in the private sdk"